build.gradle

```diff
@@ -2,7 +2,7 @@ import edu.sc.seis.launch4j.tasks.Launch4jLibraryTask
 
 plugins {
     id 'edu.sc.seis.launch4j' version '2.4.6'
-    id 'com.github.johnrengelman.shadow' version '5.2.0'
```

Comment:
Updating major versions of packages is generally something that would be done on its own as it quite often will break things in unexpected ways.

This is probably fine but it is best to do updates as their own piece of work.

Reply:
I think this is a fix I used the first time I wanted to compile because I got an error, but i think it's ok now.

Reply:
I'll update to 6 in a separate commit, might as well be on the latest